src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,17 @@
+import type { ComponentProps, FC } from 'react';
+import { BannerCollapseButton } from './BannerCollapseButton';
+
+export type BannerComponentProps = ComponentProps<'div'>;
+
+const BannerComponent: FC<BannerComponentProps> = ({ children, ...props }) => {
+  return (
+    <div data-testid="flowbite-banner" tabIndex={-1} {...props}>
```

Comment:
Add a `role="banner"` here for screen readers

Reply:
Thank you, doing this now. Also, this isn't in the vanilla library @zoltanszogyenyi